root/usr/share/fdi/facts/ethtool.rb

```diff
@@ -9,7 +9,8 @@
 require 'facter'
 
 Facter.add(:ethtool, :timeout => 10) do
-  confine :kernel => "Linux"
+  confine kernel: "Linux"
+  confine networking { |value| value && value['interfaces'] }
```

Comment:
I think this should be a symbol
```suggestion
  confine :networking { |value| value && value['interfaces'] }
```

Reply:
Hah I had to scratch my head, Ruby syntax can be confusing. It happened because I was converting the hashrocket above.

```
[lzap@nuc foreman-discovery-image]$ FACTERLIB=root/usr/share/fdi/facts facter --debug --trace ethtool
[2021-04-29 16:12:19.015287 ] INFO Facter - executed with command line: --debug --trace ethtool
[2021-04-29 16:12:19.015330 ] DEBUG Facter - Facter version: 4.1.1
[2021-04-29 16:12:19.015375 ] DEBUG Facter::ClassDiscoverer - There is no module named Fedora
[2021-04-29 16:12:19.016691 ] DEBUG Facter::FactLoader - Loading internal facts
[2021-04-29 16:12:19.016706 ] DEBUG Facter::FactLoader - Loading all internal facts
[2021-04-29 16:12:19.016764 ] DEBUG Facter::FactLoader - Loading custom facts
[2021-04-29 16:12:19.017686 ] DEBUG Facter::FactLoader - Loading external facts
[2021-04-29 16:12:19.017781 ] DEBUG Facter::QueryParser - User query is: ["ethtool"]
[2021-04-29 16:12:19.017795 ] DEBUG Facter::QueryParser - Query is ethtool
[2021-04-29 16:12:19.017810 ] DEBUG Facter::QueryParser - Checking query tokens ethtool
[2021-04-29 16:12:19.018408 ] DEBUG Facter::QueryParser - List of resolvable facts: []
[2021-04-29 16:12:19.018440 ] DEBUG Facter::InternalFactManager - Resolving facts sequentially
[2021-04-29 16:12:19.018512 ] DEBUG Facter::LegacyFactFormatter - Formatting for single user query
[2021-04-29 16:12:19.018547 ] DEBUG Facter::LegacyFactFormatter - Remove quotes from value if it is a simple string
```